packages/git/src/node/dugite-git-watcher.ts

```diff
@@ -28,19 +29,31 @@ export class DugiteGitWatcherServer implements GitWatcherServer {
             watcher.dispose();
         }
         this.watchers.clear();
+        this.subscriptions.clear();
```

Comment:
We should iterator over the subscriptions here and call dispose ?

Reply:
OK.

Reply:
it should not be necessary, they should be removed on `watcher.dispose()`call.

Reply:
haa OK yes indeed.